Add linux kernel benchmark

PappasBrent opened the pull request:
Adds a script for running `vast-front` on the Linux kernel benchmark, along with instructions for setting up the benchmark and running the script.

PappasBrent committed:
bench: add Linux kernel benchmark script

PappasBrent committed:
bench: Add Linux kernel benchmark instructions

Add instructions for setting up and running the Linux kernel benchmark.

Jezurko commented on benchmarks/linux_kernel/linux_testing_guide.md:
```diff
@@ -0,0 +1,86 @@
+
+## Foreword
+
+These instructions were tested and written on an `XPS 13 9310` machine running
+`Ubuntu 22.04.4 LTS x86_64`.
```
I would leave out the machine name, just mention the Ubuntu version, as that might be useful for reference/docker instances.

Jezurko reviewed (approved):
Looks good to me.
Do you think you could also add some small script that reads the `tsv` and outputs a markdown file? We use those to put the benchmark results on the VAST website.
Ideally make it so that the user can choose the separator (so ti can also be `csv` or whatever). It will most likely be useful with other benchmarks in the future as well, so it can be in some `utils` or `tools` folder.
You can use `pandas` for it, which should make it pretty simple - `read_csv` and then dump the obtained `dataframe` using `to_markdown`.

PappasBrent commented:
Sure I can add the converter

PappasBrent committed:
bench: remove machine information

Remove machine information from instructions and just leave operating
system information.

PappasBrent committed:
bench: Print passing compilation units to stderr

Print total passing compilation units to stderr instead of stdout so
that other scripts can process TSV output directly without needing to
remove the last line of output.

PappasBrent committed:
util: add script to convert field-separated values

Add script to convert a series of field-separated values to markdown
table
